tests/integration/external/guzzle7/test_cat.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Test that CAT works with guzzle 7.
+*/
+
+/*SKIPIF
+<?php
+require_once(realpath(dirname(__FILE__)) . '/../../../include/unpack_guzzle.php');
+
+if (version_compare(phpversion(), '7.2.0', '<=')) {
+    die("skip: PHP > 7.2.0 required\n");
+}
+
+if (!unpack_guzzle(7)) {
+    die("skip: guzzle 7 installation required\n");
+}
+
+if (!isset($_ENV["ACCOUNT_supportability"]) || !isset($_ENV["APP_supportability"])) {
+    die("skip: env vars required");
+}
+*/
+
+/*INI
+newrelic.distributed_tracing_enabled=0
+newrelic.cross_application_tracer.enabled = true
+*/
+
+/*EXPECT
+tracing endpoint reached
+tracing endpoint reached
+Customer-Header=found tracing endpoint reached
+*/
+
+/*EXPECT_RESPONSE_HEADERS
+X-NewRelic-App-Data=??
+X-NewRelic-App-Data=??
+X-NewRelic-App-Data=??
+*/
+
+/*EXPECT_METRICS
+[
+  "?? agent run id",
+  "?? timeframe start",
+  "?? timeframe stop",
+  [
+    [{"name":"External/127.0.0.1/all"},                   [3, "??", "??", "??", "??", "??"]],
+    [{"name":"External/all"},                             [3, "??", "??", "??", "??", "??"]],
+    [{"name":"External/allOther"},                        [3, "??", "??", "??", "??", "??"]],
+    [{"name":"ExternalApp/127.0.0.1/ENV[ACCOUNT_supportability]#ENV[APP_supportability]/all"},
+                                                          [3, "??", "??", "??", "??", "??"]],
+    [{"name":"ExternalTransaction/127.0.0.1/ENV[ACCOUNT_supportability]#ENV[APP_supportability]/WebTransaction/Custom/tracing"},
+                                                          [3, "??", "??", "??", "??", "??"]],
+    [{"name":"ExternalTransaction/127.0.0.1/ENV[ACCOUNT_supportability]#ENV[APP_supportability]/WebTransaction/Custom/tracing",
+      "scope":"OtherTransaction/php__FILE__"},            [3, "??", "??", "??", "??", "??"]],
+    [{"name":"OtherTransaction/all"},                     [1, "??", "??", "??", "??", "??"]],
+    [{"name":"OtherTransaction/php__FILE__"},             [1, "??", "??", "??", "??", "??"]],
+    [{"name":"OtherTransactionTotalTime"},                [1, "??", "??", "??", "??", "??"]],
+    [{"name":"OtherTransactionTotalTime/php__FILE__"},    [1, "??", "??", "??", "??", "??"]],
+    [{"name":"Supportability/library/Guzzle 4-5/detected"},
+                                                          [2,    0,    0,    0,    0,    0]],
```

Comment:
Why is this a 2?

Reply:
Thanks for catching that! I need to look further into this and figure out why the program outputs a 2 for guzzle 4-5 detection. Guzzle 7 is being correctly detected, which is good since this is a guzzle 7 test case.

Reply:
Hi Zach, I figured out that it was outputting a 2 for Guzzle 4-5 because the file path "clientinterface.php" was being detected twice inside of the guzzle 7 package. This is why I have updated the file path for each guzzle version to ensure that each file path is unique, so that we only detect the correct guzzle version based on which test is being run. [48f8e9e](https://github.com/newrelic/newrelic-php-agent/pull/498/commits/48f8e9e98d1a663c0af3b351a7a7219873e157b4)!